cmd/root.go

```diff
@@ -171,11 +171,11 @@ func startConsumer(queue Queue) {
 	for msg := range queue.sem {
 		if sendMessageToURL(*msg.Body, queue) {
 			_, err := svc.DeleteMessage(&sqs.DeleteMessageInput{
-				QueueUrl:      &queue.URL,
+				QueueUrl:      queue.msgparams.QueueUrl,
```

Comment:
Why is this change?

Reply:
@ameykpatil this was a bug. QueueURL needs to be SQS queue URL, I had used URL it's supposed to hit by mistake.